src/modifierGolemancer.ts

```diff
@@ -13,7 +13,7 @@ import { UnitType } from "./types/commonTypes";
 import { makeRisingParticles } from "./graphics/ParticleCollection";
 import { BLOOD_GOLEM_ID } from "./entity/units/bloodGolem";
 
-// Summon up to [quantity] golems each turn
+// Summon up to [quantity] golems each turn and grants the player summon golem spell
```

Comment:
I still feel like this would work better as "summons X golems at the start of each turn" rather than topping off the player's number of ally golems.  It just feels confusing to me.
If you're dead-set on keeping it as is, we need to find a way to change the language so it's crystal clear

Reply:
Yeahh.. it might be a bit confusing. I'm not super dead set on it, but here was my reasoning in case you were curious:

One issue I have with the summon playstyle is that it's 100% centered around hitting a "critical mass" of units that can finish the level on their own. You sit in a corner and repeatedly summon things until your army is bigger than their army, and that never felt very interesting to me. 

The vision is that Golemancer gives you a handful of summons to work with at any given time, without letting you reach that critical mass. The value of each summon is immediate: they block an attack, deal some damage, serve as a bloat bomb, or turn into a corpse for later use, and you almost want to use them up quickly to achieve whatever your current goal is. The cap ensures they'll never win the round on their own, forcing you to continue strategizing and adapting to the moment, and their immediate availability encourages interaction. You can never sit back and spam "end turn" and you're not just creating a wall or critical mass of summons

If I make it spawn X each turn and remove the cap, the rune is obviously much stronger, and we'll have to drastically increase the cost to balance it. It'll make the starting squad smaller and the end squad bigger, leaning more into the setup -> tipping point playstyle. Which is fine, just not what I was initially going for

The main goal was to diversify summon builds and create some variant of necromancer that calls for a little more strategy / creates more interesting scenarios.

.

Language wise, I'm not really sure...
- Summons X respawning golems
- Summons golems each turn, up to a limit of X.
- Creates a squad of X golems. Squad golems respawn each turn
- Summons X golems each turn. Cannot summon more than X golems at once

Reply:
I'm happy for it to be implemented either way. There's definitely some value in making it more intuitive and consistent with other summon effects

Reply:
Oh I see.  I like that logic.  Let's go with:
"Creates a squad of X golems. Dead squad golems respawn each turn".

However, how does this handle if you use Summon Golem independent of the "squad"?

Reply:
I'll need to update the logic to differentiate squad and non-squad golems if we go with that description.
Right now *any ally golem* counts towards the max, so its more like

"Each turn, if you have less than X golems, summon up to X golems"
"Summon golems each turn. Minimum of X summoned golems at the start of each turn"
"Golems are automatically summoned each turn if you have less than X golems"

Reply:
Or more simply:
"Creates enough ally golems to ensure you have X at the start of each turn"

Reply:
Yeah, that works!